src/components/editions/headline.tsx

```diff
@@ -53,9 +53,8 @@ const styles = (format: Format): SerializedStyles => css`
 	box-sizing: border-box;
 	border-top: 1px solid ${border.secondary};
 	padding-bottom: ${remSpace[4]};
+	padding-right: ${remSpace[2]};
 	margin: 0;
-
-	${articleWidthStyles}
```

Comment:
This breaks the gallery headline width

Reply:
Adding it to `galleryStyles` the way you've added it to interviewStyles should fix this.

Reply:
Done.